.github/workflows/ci.yml

```diff
@@ -1,17 +1,16 @@
+---
```

Comment:
GitHub Actions workflows don't support the entire YAML spec nor multidocument YAML files, so this is entirely superfluous.  
Please read the documentation about the workflow syntax: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions

Not even anchors and I wish that wasn't so.

Reply:
may be, but prefer to use yamllint, and no warnings!

Reply:
Like I said, multidocument YAML files are not supported by GitHub Actions and this project doesn't use YAMLlint.